src/mixins/stackable.js

```diff
@@ -42,7 +42,9 @@ export default {
       // Convert the NodeList to an array to
       // prevent an Edge bug with Symbol.iterator
       // https://github.com/vuetifyjs/vuetify/issues/2146
-      const activeElements = [...document.getElementsByClassName(this.stackClass)]
+      const activeElements = typeof document !== 'undefined' 
```

Comment:
extra space at the end, should of been caught by linter, did you push with --no-verify?

Reply:
that change was made directly on github :)